pom.xml

```diff
@@ -934,12 +934,17 @@
                     <excludes>
                         <exclude>org/apache/druid/java/util/common/DateTimes$UtcFormatter.class</exclude>
                         <exclude>org/apache/druid/java/util/common/DateTimes.class</exclude>
+                        <exclude>org/apache/druid/cli/PullDependencies.class</exclude>
```

Comment:
Instead of excluding whole files, could you annotate methods `@SuppressForbidden`

Reply:
Nice suggestion. Done.